docs/data/pages.ts

```diff
@@ -278,6 +278,10 @@ const pages: MuiPage[] = [
         subheader: 'Upgrade to v6',
         children: [
           { pathname: '/x/migration/migration-data-grid-v5', title: 'Breaking changes: Data Grid' },
+          {
+            pathname: '/x/migration/migration-pickers-v5',
+            title: 'Breaking changes: Date & Time Pickers',
```

Comment:
We creating those packages, we decided to go with the `Date and time pickers` wording to describe the new components.
The capitalization was not clearly defined (and with the recent decision we could unify everywhere).

I wanted to avoid `Pickers` because if one day we have a color picker, or an emoji picker, they would not be in the scope of those packages.

For the NPM packages, we favored concision by dropping the `-and-time` part.
I agree that this is inconsistent. I would not rename the NPM package even if we all thought that another name would be better, because it's a very heavy change that will bother our community.

If we do want to change the naming used in the doc and in the blog posts, I would be in favor of thinking it thoroughly and have @samuelsycamore in the loop.


_____

For the smaller change (`Breaking changes: Date and Time pickers` insteand of `Date & Time pickers`), I fully agree, it's a typo on my side.